code/src/main/java/com/google/sps/storage/data-storage/Business.java

```diff
@@ -0,0 +1,41 @@
+package com.google.sps.storage;
+
+import com.googlecode.objectify.annotation.Entity;
+import com.googlecode.objectify.annotation.Id;
+import java.time.Instant;
+import java.util.Optional;
+
+/**
+* This class is used to check if a business was contacted in the last week.
+*/
+@Entity
+public class Business {
+
+  @Id private String placeId;
```

Comment:
Also, I'm not sure if this has some issue with index, but a lot of these look to just be stores of value. If the values can't change it is worth making then final.

Reply:
I agree, this might be my lack of Java experience, but I was not sure if making them final would automatically set them to null and make it so we cannot change the value?

Reply:
I just looked up and it says final variables can be assigned once if they are left blank

Reply:
However, since Objectify requires a constructor with no parameters we would have to set placeId inside that empty constructor?

Reply:
Hmm. I think then, unless we can figure out what Objectify is doing there and work around it, it's probably best to only make the variables you can final, ones that need to be passed in might necessarily be mutable.